### PR TITLE
Tkt 00199 - information card

### PR DESCRIPTION
**Pull request by Android63**
MVP information card without media or accordion dropdown for now.

screen shot 2018-04-09 at 11 36 36

### CODE REVIEW

**Comment by Android63** [comment]
Please note that, since the markup and styling is almost identical for alerts and information cards, I have merged #200 into #199

File: components/paragraphs/_components.paragraphs.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* ==========================================================================
+   #PARAGRAPHS
+   ========================================================================== */
+
+/**
+ * Smaller font size for supporting paragraphs in alert boxes, etc.
+ *
+ */
+
+.c-paragraph--supporting {
```
Comment (cehwitham) [inline components/paragraphs/_components.paragraphs.scss]: @Android63 is this used in multiple places or should this be achieved with a Mix?

**Review by cehwitham — COMMENTED** [review]
Feels like there's a bit of confusion with the paragraph component and we just need to clear up how we want to implement that?

**Comment by Android63** [comment]
@cehwitham I've simplified this by using the <small> element rather than a distinct supporting paragraph style. Could you review again please?

File: settings/_settings.definitions.scss [hunk 2]
```diff
@@ -43,12 +43,12 @@ $global-font-size-leading-paragraph: 24px !default;
 $global-line-height-leading-paragraph: 32px !default;
 
 
-
 // Colors
 // ========================================================================== */
 
 $global-colors: (
   'attention': #7c2855,
+  'alert': #DA291C,
```
Comment (cehwitham) [inline settings/_settings.definitions.scss]: Watch capitalisation in values.

**Review by cehwitham — CHANGES_REQUESTED** [review]
Much better, just a small tweak needed.

**Comment by Android63** [comment]
@cehwitham I've fixed the capitalisation but I wonder if we need $color-alert at all since it's identical to $color-error. What do you think @Pro-Paul?

**Comment by penpadstudio** [comment]
@Android63 We will need to review what error, alert states we have and what level of urgency they are so we can then colour code them moving forward. But for now I think we can just stick with the initial alert and should we need to expand that we can. @cehwitham would you agree?

**Comment by Android63** [comment]
So Use same colour for both alerts and warnings.

**Comment by Android63** [comment]
@cehwitham could you review this again please?

**Review by cehwitham — APPROVED** [review]

**Pull request merged**